src/pages/Login.tsx

```diff
@@ -1,14 +1,50 @@
-import React from 'react';
+import React, { useEffect } from 'react';
+import { useDispatch } from 'react-redux';
 import { LinkFacade } from '../lib';
+import { Container } from '../components/Container';
+import { REACT_APP_API_BASE_URL, REACT_APP_GOOGLE_CLIENT_ID } from '../data';
+import { useGoogleAuth } from '../hooks';
+
+declare global {
+  interface Window {
+    google: typeof import('google-one-tap');
+  }
+}
 
 export function Login() {
+  const { handleGoogleLogin, error, loading } = useGoogleAuth();
+  const dispatch = useDispatch();
+  useEffect(() => {
+    if (window.google) {
+      window?.google.accounts.id.initialize({
+        client_id: REACT_APP_GOOGLE_CLIENT_ID as string,
+        callback: handleGoogleLogin,
+      });
+
+      window.google.accounts.id.renderButton(
+        document.getElementById('loginDiv') as HTMLDivElement,
+        {
+          theme: 'outline',
+          text: 'continue_with',
+          shape: 'pill',
+          type: 'standard',
+          logo_alignment: 'center',
+        },
+      );
+    }
+  }, [dispatch, handleGoogleLogin]);
+
   return (
-    <div>
+    <Container>
       <h1>Login</h1>
       <p>Awesome 🎉! You just reached the login page.</p>
-
+      <p>{REACT_APP_API_BASE_URL}</p>
       <p>Now try to navigate to the home page.</p>
       <LinkFacade to="/">Home</LinkFacade>
-    </div>
+      <main className="flex__center">
+        {error && <p style={{ color: 'red' }}>{error}</p>}
+        {loading ? <div>Loading....</div> : <div id="loginDiv" />}
```

Comment:
@musllim 
Login will still be displayed even if there is an error. 
```suggestion
        {!error && loading ? <div>Loading....</div> : <div id="loginDiv" />}
```